minecraft_servers/playlegend/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+    "server_name": "playlegend",
+    "nice_name": "PlayLegend",
+    "direct_ip": "playlegend.net",
+    "server_wildcards": [
+      "%.playlegend.net"
+    ],
+    "supported_languages": [
+       "en",
+       "de",
+       "fr"
+    ],
+    "social": {
+      "web": "https://playlegend.net",
+      "web_shop": "https://shop.playlegend.net",
+      "web_support": "https://playlegend.net",
```

Comment:
web_support cannot be just plain website; needs to be FAQ, contact form or similar things